app/models/state_file_archived_intake_request.rb

```diff
@@ -15,19 +17,80 @@
 #
 #  idx_on_state_file_archived_intake_id_7dd0f99380  (state_file_archived_intake_id)
 #
+require 'csv'
+
 class StateFileArchivedIntakeRequest < ApplicationRecord
   devise :lockable, unlock_in: 60.minutes, unlock_strategy: :time
   has_many :state_file_archived_intake_access_logs, class_name: 'StateFileArchivedIntakeAccessLog'
   belongs_to :state_file_archived_intake, class_name: 'StateFileArchivedIntake', optional: true
 
+  before_create :populate_fake_addresses
+
   def self.maximum_attempts
     2
   end
 
   def increment_failed_attempts
     super
-    if attempts_exceeded? && !access_locked?
-      lock_access!
+    lock_access! if attempts_exceeded? && !access_locked?
+  end
+
+  def fake_addresses
+    [fake_address_1, fake_address_2]
+  end
+
+  private
+
+  def populate_fake_addresses
+    self.fake_address_1, self.fake_address_2 = fetch_random_addresses
```

Comment:
also sorry if this is obvious, and i missed it, how do you determine if the address if NY or AZ?

Reply:
On line 66. There is a NY and an AZ csv.